draft-ietf-tls-deprecate-obsolete-kex.md

```diff
@@ -259,40 +251,21 @@ Given these problems, this document updates {{!RFC4346}}, {{!RFC5246}},
 
 # Non-Ephemeral Diffie Hellman {#non-ephemeral}
 
-Clients MUST NOT offer non-ephemeral FFDH cipher suites in TLS 1.2 connections.
+Clients and servers MUST NOT offer non-ephemeral FFDH cipher suites in TLS 1.2 connections.
 (Note that TLS 1.0 and 1.1 are deprecated by {{!RFC8996}} and TLS 1.3 does not
 support FFDH {{!RFC8446}}.) This includes all cipher suites listed in the table in
 {{appendix-dh}}.
 
-Clients SHOULD NOT offer non-ephemeral ECDH cipher suites in TLS 1.2
+Clients and servers SHOULD NOT offer non-ephemeral ECDH cipher suites in TLS 1.2
 connections. (Note that TLS 1.0 and 1.1 are deprecated by {{!RFC8996}} and
 TLS 1.3 does not support ECDH {{!RFC8446}}.) This includes all cipher suites listed
 in the table in {{appendix-ecdh}}.
 
 # Ephemeral Finite Field Diffie Hellman {#dhe}
 
-Clients and servers MAY offer fully ephemeral FFDHE cipher suites in TLS 1.2
-connections under the following conditions:
-
-1. Clients and servers MUST NOT reuse ephemeral DHE public keys across TLS
-connections for all existing (and future) TLS versions. Doing so invalidates
-forward secrecy properties of these connections. For DHE, such reuse may also
-lead to vulnerabilities such as those used in the {{Raccoon}} attack. See
-{{sec-considerations}} for related discussion.
-
-2. The group size is at least 2048 bits.
-
-(Note that TLS 1.0 and 1.1 are deprecated by {{!RFC8996}}. TLS 1.3 satisfies the
-second point above {{!RFC8446}} and is not vulnerable to the {{Raccoon}} Attack.)
-
-We note that, previously, supporting the broadest range of clients would have
-required supporting either RSA key exchange or 1024-bit FFDHE. This is no longer
-the case, and it is possible to support most clients released since circa 2015
-using 2048-bit FFDHE or more modern key exchange methods, and without RSA key
-exchange {{server_side_tls}}.
-
-All the cipher suites that do not meet the above requirements are listed in the
-table in {{appendix-dhe}}.
+Clients and servers MUST NOT offer FFDHE cipher suites in TLS 1.2 connections.
+This includes all cipher suites listed in the table in {{appendix-dhe}}.
+(Note that TLS 1.0 and 1.1 are deprecated by {{!RFC8996}}.) FFDHE cipher suites in TLS 1.3 do not suffer from the above problems {{!RFC8446}}. Therefore, clients and servers MAY offer FFDHE cipher suites in TLS 1.3 connections
```

Comment:
Nit: There seems to be a missing period at the end of this sentence.

Reply:
Done, thanks :-)